server/migrations/postgres/20240422191813_create-files.up.sql

```diff
@@ -0,0 +1,8 @@
+create table files (
+    id uuid primary key,
+    name text not null,
+    path text unique not null,
+    size bigint not null,
+    user_id uuid,
+    constraint fk_user_id foreign key(user_id) references users(id) on delete cascade
```

Comment:
Are we sure we want to delete all files of a user if we delete the user?

Deleting files is nearly always a bad thing, this could lead to dangerous data loss. Maybe we should just orphan the files, then have a manual garbage collection system so that admins can cleanup dead files?

Reply:
No strong opinion on this, if you think that being a soft-delete is a better option and a possible feature, I can remove the delete on cascade